internal/controller/standalone_pgadmin/pod.go

```diff
@@ -245,5 +299,48 @@ done
 
 	wrapper := `monitor() {` + startScript + reloadScript + `}; export cluster_file="$1"; export -f monitor; exec -a "$0" bash -ceu monitor`
 
-	return []string{"bash", "-ceu", "--", wrapper, "pgadmin", fmt.Sprintf("%s/~postgres-operator/%s", configMountPath, settingsClusterMapKey)}
+	return []string{"bash", "-ceu", "--", wrapper, "pgadmin", fmt.Sprintf("%s/conf.d/~postgres-operator/%s", configMountPath, settingsClusterMapKey)}
```

Comment:
🏕️ This isn't something you introduced, but this `configMountPath + conf.d/~postgres-operator + clusterMap` occurs multiple times as well. It should be a constant.

⛏️ `settingsClusterMapKey` is a Secret/ConfigMap _key_, but here it is being used as a file _path_. The names should be clear about what they mean or are for. Here's an example:

https://github.com/CrunchyData/postgres-operator/blob/b05eb5433b6aa80da380926ef99935de38eb945b/internal/pgadmin/config.go#L60-L64